mappings/net/minecraft/sortme/IMultiThreadedObject.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS none/qp net/minecraft/sortme/IGameThread
+CLASS none/qp net/minecraft/sortme/IMultiThreadedObject
```

Comment:
For reference this is an interface that lets you check if you're on the main thread as well as submit tasks to the main thread.